README.md

```diff
@@ -307,6 +307,9 @@ Usage: gptme [OPTIONS] [PROMPTS]...
     /help         Show this help message
     /exit         Exit the program
 
+  Keyboard shortcuts:
+    Ctrl+J        Insert a new line without executing the prompt
+
```

Comment:
This should be added to the actual output of `gptme --help` (which is what is used to generate the README)